src/scss/components/_video.scss

```diff
@@ -0,0 +1,63 @@
+* {
```

Comment:
don't use this selector. because it affects all

Reply:
Done...